src/components/Header/index.js

```diff
@@ -1,17 +1,88 @@
-import React from "react";
-import "./Header.css";
+import React from 'react';
+import './Header.css';
+import { NavHashLink as NavLink } from 'react-router-hash-link';
 
-const Header = ({
+const Header = ({}) => {
+  const handleClick = (event) => {
+    event.target.classList.toggle('active');
+    console.log(event.target);
+  };
 
-                }) => {
   return (
     <div className="Header">
-      {/*<div className='Header-Logo'>*/}
-      {/*Logo*/}
-      {/*</div>*/}
-      {/*<div className='Header-Menu'>*/}
-      {/*fd fg dfgfd g*/}
-      {/*</div>*/}
+      <ul className="Header-Navigation" onClick={handleClick}>
+        <li>
+          <NavLink
+            to="/#about"
+            smooth={true}
+            isActive={() => {
+              return (
+                window.location.pathname + window.location.hash === '/#about'
```

Comment:
Imo you should turn this into a function, e.g.

```javascript
const isNavLinkActive = (path) => `${window.location.pathname}${window.location.hash}` === path;
```

and then use it on the `NavLink`s,

```javascript
<NavLink
  to="/#about"
  smooth={true}
  isActive={isNavLinkActive('/#about')}
/>
```